scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -0,0 +1,285 @@
+  job_openings = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.2.JobOpenings", sep="\t", expected_cols=expected_job_columns)
+  job_hires = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.3.Hires", sep="\t", expected_cols=expected_job_columns)
+  total_seps = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.4.TotalSeparations", sep="\t", expected_cols=expected_job_columns)
+  total_quits = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.5.Quits", sep="\t", expected_cols=expected_job_columns)
+  total_layoffs = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.6.LayoffsDischarges", sep="\t", expected_cols=expected_job_columns)
+  total_other_seps = web_to_pandas("https://download.bls.gov/pub/time.series/jt/jt.data.7.OtherSeparations", sep="\t", expected_cols=expected_job_columns)
+
+  # Additional information about each dataframe
+  # Tuple Format: Statistical Variable name, Stat Var population, Stat Var Job Change Type If Relevant, Dataframe for Stat Var 
+  schema_mapping = [
+    ("NumJobOpening", "schema:JobPosting", "", job_openings),
+    ("NumJobHire", "dcs:BLSWorker", "Hire", job_hires),
+    ("NumSeparation", "dcs:BLSWorker", "Separation", total_seps),
+    ("NumVoluntarySeparation", "dcs:BLSWorker", "VoluntarySeparation", total_quits),
+    ("NumInvoluntarySeparation", "dcs:BLSWorker", "InvoluntarySeparation", total_layoffs),
+    ("NumOtherSeparation", "dcs:BLSWorker", "OtherSeparation", total_other_seps),
+  ]
+
+  # Combine datasets into a single dataframe including origin of data
+  jolts_df = pd.DataFrame()
+  columns_to_keep = ['series_id', 'year', 'period', 'value', 'footnote_codes']
+
+  for schema_name, population_type, job_change_event, df in schema_mapping:
+    df = df.loc[:, columns_to_keep]
+    df['statistical_variable'] = schema_name
+    df['job_change_event'] = job_change_event
+    df['population_type'] = population_type
+    jolts_df = jolts_df.append(df)
+
+  # Drop non-monthly data
+  jolts_df = jolts_df.query("period != 'M13'")
+
+  # Change date to ISO Format (YYYY-MM)
+  def period_year_to_iso_8601(row):
+    month = row['period'].lstrip("M")
+    year = row['year']
+    return f"{year}-{month}"
+
+  jolts_df.loc[:, 'Date'] = jolts_df.apply(period_year_to_iso_8601, axis=1)
+
+  # Add relevant columns from series information
+  jolts_df = jolts_df.merge(series_desc[['industry_code', 'region_code', 'seasonal', 'ratelevel_code']], left_on=["series_id"], right_index=True)
+
+  # Drop rate data
+  jolts_df = jolts_df.query("ratelevel_code == 'L'")
+
+  # Drop prelimary data
+  jolts_df = jolts_df.query("footnote_codes != 'P'")
+
+  # Drop non-national data
+  jolts_df = jolts_df.query("region_code == '00'")
+
+  # Map industries
+  def jolts_code_map(row):
+    """ Maps industry code used by BLS Jolts to NAICS or BLS aggregation """
+    industry_code = row['industry_code']
+    assert industry_code in CODE_MAPPINGS
+
```

Comment:
Let's remove this blank line similar to https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace.